include/pros/motors.h

```diff
@@ -597,11 +597,18 @@ typedef enum motor_encoder_units_e {
  */
 typedef enum motor_gearset_e {
 	E_MOTOR_GEARSET_36 = 0,  // 36:1, 100 RPM, Red gear set
+	E_MOTOR_GEAR_RED = 0,
+	E_MOTOR_GEAR_100 = 0,
 	E_MOTOR_GEARSET_18 = 1,  // 18:1, 200 RPM, Green gear set
+	E_MOTOR_GEAR_GREEN = 1,
+	E_MOTOR_GEAR_200 = 1,
 	E_MOTOR_GEARSET_06 = 2,  // 6:1, 600 RPM, Blue gear set
+	E_MOTOR_GEAR_BLUE  = 2,
+	E_MOTOR_GEAR_600 = 2,
```

Comment:
```suggestion
	E_MOTOR_GEARSET_36 = 0,  // 36:1, 100 RPM, Red gear set
	E_MOTOR_GEAR_RED = E_MOTOR_GEARSET_36,
	E_MOTOR_GEAR_100 = E_MOTOR_GEARSET_36,
	E_MOTOR_GEARSET_18 = 1,  // 18:1, 200 RPM, Green gear set
	E_MOTOR_GEAR_GREEN = E_MOTOR_GEARSET_18,
	E_MOTOR_GEAR_200 = E_MOTOR_GEARSET_18,
	E_MOTOR_GEARSET_06 = 2,  // 6:1, 600 RPM, Blue gear set
	E_MOTOR_GEAR_BLUE  = E_MOTOR_GEARSET_06,
	E_MOTOR_GEAR_600 = E_MOTOR_GEARSET_06,
```

makes maintenance slightly easier